src/resmom/stage_func.c

```diff
@@ -1061,8 +1069,12 @@ stage_file(int dir, int	rmtflag, char *owner, struct rqfpair *pair, int conn, cp
 		DBPRT(("%s: simple copy, no wildcards\n", __func__))
 		rc = copy_file(dir, rmtflag, owner, source,
 			pair, conn, stage_inout, prmt);
-		if (rc != 0)
+		if (rc != 0) {
+			snprintf(log_buffer, sizeof(log_buffer), "%s stage%s failed for %s directory owner=%s, src=%s",
+					 (rmtflag == 1) ? "remote" : "local", (dir == STAGE_DIR_OUT) ? "out" : "in", dname, owner, source);
+			log_err(-1, __func__, log_buffer);
```

Comment:
Same suggestion as above for passing errno instead of -1.

Reply:
Logging prmt (path to destination if stageout else source path) will be useful